packages/bridges/slack/README.ja.md

```
@@ -0,0 +1,151 @@
+
+## セッション粒度（新機能）
+
+> **「セッション」とは?** MulmoClaude では、1つの *セッション* = AI との 1 本の会話履歴です。AI はそのセッション内で過去の発言を覚えていて、それを踏まえて返事します。以下の設定は **1 つの Slack チャネルが何個のセッションに分割されるか** を決めます。
+
+`SLACK_SESSION_GRANULARITY` 環境変数で 3 モードから選べます:
+
+### 🗂 `channel`（デフォルト）— チャネルあたり 1 セッション
+
+`#ai-help` の中のすべての発言が **1 本の長い会話** になります。誰が話しても、スレッドを使っていても、全部同じ会話扱いです。
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)